editor.planx.uk/src/@planx/components/Pay/Public/FeeBreakdown/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface FeeBreakdown {
+  applicationFee: number;
+  total: number;
```

Comment:
nit: introducing `applicationFee` and `total` language feels like a bit of extra mental load here and I wonder if this can re-use `calculated` (which if undefined, fallsback to payable) and `payable` instead?

Reply:
Before reading the tests, I would have also personally guessed that `total` = full `calculated` amount and `applicationFee` = `payable` but your passport mappings are the other way around !

Reply:
Useful feedback thanks - this was an intentional decision to try and make it a little clearer and more explicit (I did also consider gross/net as well at one point). 

Clearly not successful though 😂 Your point about consistency and mental load is a very valid one 👍